src/diff-snapshot.js

```diff
@@ -70,7 +127,9 @@ function diffImageToSnapshot(options) {
     const diffRatio = diffPixelCount / totalPixels;
 
     let pass = false;
-    if (failureThresholdType === 'pixel') {
+    if (hasSizeMismatch) {
+      // Always fail test on image size mismatch
```

Comment:
Should we explicitly set `pass` to `false` for clarity of what the intent is?

Reply:
Alright, done